src/esp/physics/bullet/BulletRigidObject.cpp

```diff
@@ -220,6 +243,30 @@ bool BulletRigidObject::initializeObject(
   return true;
 }
 
+void BulletRigidObject::setCollisionFromBB() {
+  btVector3 dim(cumulativeBB_.size() / 2.0);
+  bObjectShape_->removeChildShape(bGenericShapes_.back().get());
+  bGenericShapes_.clear();
+  bGenericShapes_.emplace_back(std::make_unique<btBoxShape>(dim));
+  btTransform t;
+  t.setIdentity();
+  bObjectShape_->addChildShape(t, bGenericShapes_.back().get());
+  bObjectShape_->recalculateLocalAabb();
+  bObjectRigidBody_->setCollisionShape(bObjectShape_.get());
+
+  if (bObjectRigidBody_->getInvInertiaDiagLocal() == btVector3{0, 0, 0}) {
+    btVector3 bInertia(getInertiaVector());
+    // allow bullet to compute the inertia tensor if we don't have one
+    bObjectShape_->calculateLocalInertia(getMass(),
+                                         bInertia);  // overrides bInertia
+
+    LOG(INFO) << "Automatic BB object inertia computed: " << bInertia.x() << " "
+              << bInertia.y() << " " << bInertia.z();
+
+    setInertiaVector(Magnum::Vector3(bInertia));
+  }
+}
```

Comment:
The logic above is now duplicated. Could it be put into its own function?

Reply:
These two instances are different enough that this wouldn't give us much. Since one is before initialization of the bObjectRigidBody_ and the other is after, most of the get/set code would still need to stay for each.